torch/distributed/_shard/api.py

```diff
@@ -118,3 +119,15 @@ def shard_parameter(
 
     # Now we can set the attribute appropriately.
     setattr(module, param_name, st)
+
+
+def _replicate_tensor(tensor: torch.Tensor) -> ReplicatedTensor:
```

Comment:
Do we really need an API like this? Users can just call `ReplicatedTensor(tensor)`?

Reply:
My original intention on introducing this API is to make sure we have consistent APIs provided to the user. We now have:
1. shard_module(module,plan)
2. shard_parameter(module, param_name, spec)
3. _shard_tensor(tensor, spec)
All these three APIs are being used to mark a param or tensor as ShardedTensor, I feel that we should have a similar API to mark a tensor as ReplicatedTensor, it makes the API more consistent from user prospective. Let me know if this make sense or we could just use `ReplicatedTensor(tensor)`